doc/03_reference/instruction_decode_execute.rst

```diff
@@ -97,8 +97,6 @@ Bit Manipulation Extension
   +---------------------------------+---------------+--------------------------+
   | Zbt (Ternary)                   | Balanced/Full | All                      |
   +---------------------------------+---------------+--------------------------+
-  | Zb_tmp (Temporary) [#B_zb_tmp]_ | Balanced/Full | None                     |
-  +---------------------------------+---------------+--------------------------+
```

Comment:
Do these changes mean that we need to drop the kGE estimates for the different flavours below?

Reply:
Yes, indeed. I will update the Yosys numbers once the bitmanip work is completed.